mpolyakov/homework_5/Makefile

```diff
@@ -0,0 +1,19 @@
+test:
+	@pytest tests
+
+test_cov:
+	@pytest --cov-config=.coveragerc --cov=. tests/
```

Comment:
does it include `venv` coverage?

Reply:
No: `[run]
omit = 
    */hw5_venv/*
    */tests/*`

Reply:
I don't see `.coveragerc` in your PR

Reply:
Fixed